README.md

```diff
@@ -47,6 +47,7 @@ name
 [mutex_atomic](https://github.com/Manishearth/rust-clippy/wiki#mutex_atomic)                             | warn    | using a Mutex where an atomic value could be used instead
 [mutex_integer](https://github.com/Manishearth/rust-clippy/wiki#mutex_integer)                           | allow   | using a Mutex for an integer type
 [needless_bool](https://github.com/Manishearth/rust-clippy/wiki#needless_bool)                           | warn    | if-statements with plain booleans in the then- and else-clause, e.g. `if p { true } else { false }`
+[needless_continue] (https://github.com/Manishearth/rust-clippy/wiki#needless_continue)                  | warn    | use of continue statements where they are not really needed.
```

Comment:
nit: there shouldn't be a space here.

(Also, this file should be autogenerated by `python utils/update_lints.py`)